src/cljs/webchange/ui_framework/layout/navigation/styles.scss

```diff
@@ -2,71 +2,74 @@
 
 $left-menu-width: 296px;
 
-.left-menu {
-    background: #353535;
-    padding: 40px 24px 88px;
-    box-sizing: border-box;
+.left-menu { 
+   background: #353535;
+   box-sizing: border-box;
+   height: 100%;
 
-    img {
-       max-width: 100%;
-       vertical-align: middle;
-    }
- 
-    .icon-top {
-       padding-left: 16px;
-       margin-bottom: 47px;
-       display: inline-block;
-    }
- 
-    .nav-menu {
-       ul,
-       ul li {
-          cursor: pointer;
-          list-style: none;
-          list-style-type: none;
-          margin: 0;
-          padding: 0;
-       }
- 
-       ul li {
-          font-weight: normal;
-          font-size: 14px;
-          line-height: 20px;
-          align-items: center;
-          color: #9e9e9e;
-          margin-bottom: 8px;
-          display: table;
-          text-decoration: none;
-          padding: 10px 18px;
-          min-width: 168px;
- 
-          span {
-             height: 20px;
-             display: inline-block;
+   .logo-icon {
+      margin: 40px 0px 66px 40px;
+   }
 
-             .user-icon {
-                padding: 0;
-             }
-          }
- 
-          span:first-child {
-             width: 20px;
-             float: left;
-          }
- 
-          span:nth-child(2) {
-             width: calc(100% - 20px);
-             padding-left: 18px;
-             float: left;
-          }
-       }
- 
-       ul li:hover,
-       .nav-menu ul li.active {
+   .nav-link-section {
+      margin: 0px 0px 0px 40px;
+
+      .nav-item-section {
+         list-style-type: none;
+         padding: 0px;
+         margin: 0px;
+         
+         li {
```

Comment:
Strange hover area for navigation items

![Selection_034](https://user-images.githubusercontent.com/15379954/132624586-0d168c3c-7016-4e25-a7e7-862ab2c1b48e.png)